SketchUp/Geom/BoundingBox.rb

```diff
@@ -253,10 +249,10 @@ def initialize
   #   boundingboxes actually overlap - then call this to get the resulting
   #   boundingbox.
   #
-  # @param boundingbox
+  # @param boundingbox [Geom::BoundingBox]
```

Comment:
But don't worry about manually correcting this. I think running another pass after merging will correct it in the stubs. I can then pull and merge that back in afterwards. (At least that's my theory right now.)